main/fixtures/package.json

```diff
@@ -1,25 +1,132 @@
+      "pkgbase": "glibc",
+      "pkgrel": "5",
+      "flag_date": null,
+      "installed_size": 38140928,
+      "packager_str": "Jan Alexander Steffens (heftig) <jan.steffens@gmail.com>",
+      "last_update": "2017-12-30T02:21:43.825Z",
+      "arch": 3,
+      "packager": null,
+      "pkgver": "2.25"
+    },
+    "model": "main.package",
+    "pk": 3
+  },
+  {
+    "fields": {
+      "build_date": "2017-06-13T12:11:40Z",
+      "compressed_size": 753348,
+      "pkgname": "pacman",
+      "files_last_update": null,
+      "pkgdesc": "A library-based package manager with dependency support",
+      "created": "2017-12-30T02:21:43.848Z",
+      "url": "http://www.archlinux.org/pacman/",
+      "epoch": 0,
+      "filename": "pacman-5.0.2-1-x86_64.pkg.tar.xz",
+      "repo": 1,
+      "signature_bytes": "iQFCBAABCAAsFiEEAv0cepNOYUVFhJ8ZpiNAdEmOnO4FAlk/1rAOHGFyY2hAZXdvcm0uZGUACgkQpiNAdEmOnO5kXAf/bfbuoh1VNU/tOjfAAaR8ahtxihvqobMfpLTIeLuZDjk0U1sEE8ltk7Ky8wwFWduz1SSjeCuSm/020zizNXrCv1Wq3x6mOANs2VY+JDbWMk48KAR9dajY1buu8rWRq+Rt0mFA8XTZhVXhijG7hXYWSVBtcokevrN3cDizxchb6gQeh0xYE7/vlHwblch+Y+MXMR0bbjF8xEIITZYr4895i8xvuJtD6Ll7Wr/WBGXdFlm9Z0LmAZlDHDusJ7cC5gr7HPhUcKg4tnm8mtVmCJugoiHUVel5llZ1MW4iUCcx4suUyQAqKa9HqxXE9e5ADzZHBdZA4hJ6UQQwDRm57cLwFQ==",
+      "pkgbase": "pacman",
+      "pkgrel": "1",
+      "flag_date": null,
+      "installed_size": 4694016,
+      "packager_str": "Christian Hesse <arch@eworm.de>",
+      "last_update": "2017-12-30T02:21:43.848Z",
+      "arch": 3,
+      "packager": null,
+      "pkgver": "5.0.2"
+    },
+    "model": "main.package",
+    "pk": 4
+  },
+  {
+    "fields": {
+      "build_date": "2017-01-28T22:32:41Z",
+      "compressed_size": 3905996,
+      "pkgname": "systemd",
+      "files_last_update": null,
+      "pkgdesc": "system and service manager",
+      "created": "2017-12-30T02:21:43.865Z",
+      "url": "https://www.github.com/systemd/systemd",
+      "epoch": 0,
+      "filename": "systemd-232-8-x86_64.pkg.tar.xz",
+      "repo": 1,
+      "signature_bytes": "iQEzBAABCAAdFiEESH6swIVXrQggiNq6HrJjj/VsDFMFAliOFSIACgkQHrJjj/VsDFPKYgf/aM3TnZOHoU4avTZt0KmPszRN9o4iQD50buvwgre3kt7WiiOqxpPgyML7B/s0erT3bgz22JD199kC6RbTn+qBqo/fBPZOywX0NhVIqgPsHvZjxuDJeehboqlePtJ4LtLG882TQxb1bsf8PBzTqmX7H8n/2lEommWwiruVo9bwOeaCh5ax9V8/a6UaEMkD4VwLUkAOQkVLvi8HZG7YbczotSMhhU9tSyH36UCUaA/xW3qiibD4WT2lZbKnbaC3JtddEfJn0hrwOqkiVjN0fDWnO3PLw/KOprZ18q4Be7j8AyYBdEUtEdZEmJUkqzaAKFp+3EzL1bq+g7S8nlf/Q929bA==",
+      "pkgbase": "systemd",
+      "pkgrel": "8",
+      "flag_date": null,
+      "installed_size": 18100224,
+      "packager_str": "Dave Reisner <dreisner@archlinux.org>",
+      "last_update": "2017-12-30T02:21:43.865Z",
+      "arch": 3,
+      "packager": null,
+      "pkgver": "232"
+    },
+    "model": "main.package",
+    "pk": 5
+  }
```

Comment:
I wrote a small script to `reporead` the `devel/fixtures/core.db.tar.gz` database and then use Django to dump the package fixture again, running it through `jq` for formatting.  Might be useful to add this to the repo as well?

Reply:
Hmm yeah, but I'm thinking that we can better get rid of the core.db.tar.gz fixture and just generate the test data.

Reply:
I think it would likely make sense to have a JSON or whatever format for specifying the same data as in the DB file, but contained in a single file that can be easily diffed and such, and then have a function à la `reporead` that would insert the data into the archweb DB.